auto-value-parcel/src/main/java/com/ryanharter/auto/value/parcel/Parcelables.java

```diff
@@ -263,9 +264,9 @@ else if (type.equals(TypeName.DOUBLE) || type.equals(TypeName.DOUBLE.box()))
     else if (type.equals(TypeName.BOOLEAN) || type.equals(TypeName.BOOLEAN.box()))
       block.add("$N.writeInt($N() ? 1 : 0)", out, property.methodName);
     else if (type.equals(PARCELABLE))
-      block.add("$N().writeToParcel($N, 0)", property.methodName, out);
+      block.add("$N().writeToParcel($N, $N)", property.methodName, out, flags);
     else if (type.equals(CHARSEQUENCE))
-      block.add("$T.writeToParcel($N(), $N, 0)", TEXTUTILS, property.methodName, out);
+      block.add("$T.writeToParcel($N(), $N, $N)", TEXTUTILS, property.methodName, out, flags);
```

Comment:
Doing it this way is going above and beyond :+1:. In reality you could have just embedded `flags` directly in the string since there's no chance of collision with anything and we're not doing collision avoidance when defining the parameter name.